### PR TITLE
[enhance](cloud) Add a policy when be abnormal, tablet delay switch be

### PR DESCRIPTION
**Pull request by deardeng**

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudTabletRebalancer.java [hunk 1]
```diff
@@ -502,7 +502,7 @@ private void completeRouteInfo() {
                     Map<String, List<Long>> clusterToBackends =
                             ((CloudReplica) replica).getClusterToBackends();
                     if (!clusterToBackends.containsKey(cluster)) {
-                        long beId = ((CloudReplica) replica).hashReplicaToBe(cluster, true);
+                        long beId = ((CloudReplica) replica).hashReplicaToBe(cluster, true, true);
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudTabletRebalancer.java:505]: add comment to (`true`, `true`)

File: fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudReplica.java [hunk 2]
```diff
@@ -216,20 +219,40 @@ private long getBackendIdImpl(String cluster) {
...
+                            Pair.of(secondaryBe, System.currentTimeMillis()));
+                } else {
+                    secondaryBe = secondaryBeAndChangeTimeWithCluster.get(clusterId).first;
+                    Long changeTimestamp = secondaryBeAndChangeTimeWithCluster.get(clusterId).second;
+                    if (System.currentTimeMillis() - changeTimestamp > Config.secondary_be_validity_seconds * 1000L) {
+                        // set secondary be to tablet after secondary_be_validity_seconds
+                        setBeToTablet(clusterId, secondaryBe);
+                        secondaryBeAndChangeTimeWithCluster.remove(clusterId);
+                    }
+                }
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudReplica.java:244]: seems unnecessary

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 3]
```diff
@@ -3030,6 +3030,12 @@ public static int metaServiceRpcRetryTimes() {
     @ConfField(mutable = true, description = {"存算分离模式下是否开启大事务提交，默认false"})
     public static boolean enable_cloud_txn_lazy_commit = false;
 
+    @ConfField(mutable = true, description = {"存算分离模式下，当tablet分布的be异常，是否立即映射tablet到新的be上，默认true"})
+    public static boolean enable_immediate_be_assign = true;
```
Comment (gavinchou) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:3034]: when should we set it false? what's the benefit and penalty?

**Comment by deardeng** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by yujun777 — APPROVED** [review]
LGTM

**Pull request merged**